src/pages/product/[[...product]].tsx

```diff
@@ -50,13 +60,14 @@ const ProductPage: NextPage<InferGetStaticPropsType<typeof getStaticProps>> = ({
 
   return (
     <Layout seo={{ title: product.seo.title, description: product.seo.description }}>
+      {productJsonLdProps && <ProductJsonLd {...productJsonLdProps} />}
       <ProductPageComponent
         component={options.component}
         options={options}
         breadcrumbs={breadcrumbs}
         product={product}
         reviewHighlights={reviewHighlights}
-        reviewList={reviewList}
+        reviewsIoReviewList={reviewsIoReviewList}
```

Comment:
I abstracted the `ReviewList` types post-transform to be identical. Note that the return type for `getTrustpilotReviewList` is the same as `getReviewList`. 

I believe they are only separate props for legacy reasons. I think it would be preferable to add a `publisher` prop to the `ReviewList` and consolidate to one `reviewList` prop. Should simplify things a bit throughout.

Reply:
Yeah, hard to imagine a case where somebody wants two expensive review service accts... If they do, they can afford to modify Penny, I suspect.